subiquity/common/apidef.py

```diff
@@ -77,6 +77,8 @@ def POST(endpoint_names: List[str]) -> None:
         class client_variant:
             def POST(variant: str) -> None:
                 """Choose the install variant - desktop/server"""
+            def GET() -> str:
+                """Get the install variant - desktop/server"""
```

Comment:
Please add wsl_setup and wsl_reconfigure.

Reply:
This is fixed.